### PR TITLE
setting's tab added

### PR DESCRIPTION
**Pull request by sergevei**

### CODE REVIEW

**Review by universezzz — COMMENTED** [review]
Review completed

File: src/containers/profile/settingsTab/styles.module.scss [hunk 1]
```diff
@@ -0,0 +1,94 @@
+@import '../../../scss/shared/breakpoints.scss';
+@import '../../../scss/shared/fonts.scss';
+@import '../../../scss/shared/colors.scss';
+@import '../../../scss/mixins/button.scss';
+@import '../../../scss/mixins/input.scss';
+
```
Comment (OlegPanasyuk) [inline src/containers/profile/settingsTab/styles.module.scss:3]: In my opinion, The best way for minify code is to move reiterative code in one file and then you can import this code by one row. Now you import it each time by five rows.

File: src/containers/profile/settingsTab/styles.module.scss [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+      border: 3px #212121 solid;
+      color: #fff;
+      cursor: pointer;
+      option{
+        background-color: #262626;
+        cursor: pointer;
+      }
+    }
+  }
+  ul{
```
Comment (Pahmurny-itechart) [inline src/containers/profile/settingsTab/styles.module.scss]: codestyle: do we want a space before curly bracket? Try to pick one and stick to it.

**Review by slavasgod — APPROVED** [review]

File: src/containers/profile/index.jsx [hunk 3]
```diff
@@ -32,17 +33,17 @@ export const Profile = () => {
 
   return (
     <section className={profile}>
```
Comment (universezzz) [inline src/containers/profile/index.jsx:45]: There is a good practise, to mark tags `<article>` `<section>` with `<h1>...<h6>.`
In `<h>` you should write description of the block and then use css property display: none if this `<h> ` isn't needed in the design of the project

File: src/containers/profile/index.jsx [hunk 4]
```diff
@@ -32,17 +33,17 @@ export const Profile = () => {
...
-      <section className={mobile_burger}>
+      <div className={mobile_burger}>
         <button onClick={() => setMenu({ marginLeft: '0' })}>
```
Comment (universezzz) [inline src/containers/profile/index.jsx]: You should use buttons attribute `type` (submit, button, reset)

File: src/containers/profile/index.jsx [hunk 5]
```diff
@@ -32,17 +33,17 @@ export const Profile = () => {
...
           <div className={photo} />
         </button>
-      </section>
-      <section className={mobile_menu} style={menus}>
-        <section className={user}>
+      </div>
+      <div className={mobile_menu} style={menus}>
+        <div className={user}>
```
Comment (universezzz) [inline src/containers/profile/index.jsx]: rewrite to the `<ul> <li>` list of items.

File: src/containers/profile/index.jsx [hunk 6]
```diff
@@ -76,39 +77,35 @@ export const Profile = () => {
...
-      <section className={menu}>
-        <section className={user}>
+      <div className={menu}>
+        <div className={user}>
           <div className={photo} />
           <div className={name}>John Doe</div>
           <div className={role}>student</div>
-        </section>
+        </div>
         <button className={getStatus === 0? 'profile_button_active profile_button': 'profile_button'} onClick={() => toggleButton(0)}>
```
Comment (universezzz) [inline src/containers/profile/index.jsx]: you should simplify `className={getStatus === 0? 'profile_button_active profile_button': 'profile_button'}`
className={`${profile_button} ${getStatus === 0 ? 'profile_button_active' : ''}`}

Comment (universezzz) [reply src/containers/profile/index.jsx]: Please, use Prettier for formating code

File: src/containers/profile/settingsTab/index.jsx [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+  } = styles;
+  return (
+    <div className={settings}>
+      <div className={localization}>
+        <select name="localization" id="">
+          <option value="" defaultValue>ENG</option>
+          <option value="" >RUS</option>
+        </select>
+      </div>
+      <button>
```
Comment (universezzz) [inline src/containers/profile/settingsTab/index.jsx:18]: button attr type

File: src/containers/profile/index.jsx (hunk 5 above)
Comment (sergevei) [reply src/containers/profile/index.jsx]: I'm use simple structure for 4 elements like this:
```
 <div>
  <button .../>
  <button .../>
  <button .../>
  <button .../>
</div>
```
This is excess code if I'll change that block like:
```
<div>
   <ul>
      <li>
        <button .../>
      </li>
      <li>
        <button .../>
      </li>
      <li>
        <button .../>
      </li>
      <li>
        <button .../>
      </li>
   </ul>
</div>
```

**Pull request merged**